src/main/java/org/opensearch/geospatial/index/mapper/xypoint/XYPoint.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.geospatial.index.mapper.xypoint;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+/**
+ * Represents a point in a 2-dimensional planar coordinate system with no range limitations
+ */
+@AllArgsConstructor
+public class XYPoint {
+    @Getter
+    private double x;
+    @Getter
+    private double y;
+
+    public XYPoint() {}
```

Comment:
can we create a @NoArgsConstructor here?

Reply:
ack